pkg/remotewrite/validate.go

```diff
@@ -37,5 +37,5 @@ func ValidateRequest(remoteWriteRequest *prometheus.WriteRequest) (string, error
 		return key, nil
 	}
 
-	return "", errors.New("request does not contain any cluster ids")
+	return "", errors.New(fmt.Sprintf("request does not contain any cluster ids: %v", remoteWriteRequest))
```

Comment:
See my comment in https://github.com/bf2fc6cc711aee1a0c2a/observability-remote-write-proxy/pull/20#discussion_r1131296727